Add 4.83.0 changelog (#6274)
hashicorp/terraform-provider-google-beta: CHANGELOG.md

## 4.83.0 (Unreleased)

DEPRECATIONS:
* secretmanager: deprecated `automatic` field on `google_secret_manager_secret`. Use `auto` instead. ([#6237](https://github.com/hashicorp/terraform-provider-google-beta/pull/6237))

FEATURES:
* **New Resource:** `google_biglake_table` ([#6205](https://github.com/hashicorp/terraform-provider-google-beta/pull/6205))
* **New Resource:** `google_data_pipeline_pipeline` ([#6236](https://github.com/hashicorp/terraform-provider-google-beta/pull/6236))
* **New Resource:** `google_dialogflow_cx_test_case` ([#6249](https://github.com/hashicorp/terraform-provider-google-beta/pull/6249))
* **New Resource:** `google_storage_insights_report_config` ([#6253](https://github.com/hashicorp/terraform-provider-google-beta/pull/6253))

> * **New Resource:** google_apigee_target_server ([#6215](https://github.com/hashicorp/terraform-provider-google-beta/pull/6215))

Reviewer: ```suggestion
* **New Resource:** `google_apigee_target_server` ([#6215](https://github.com/hashicorp/terraform-provider-google-beta/pull/6215))
```

IMPROVEMENTS:

> * GKEHUB: added `labels` fields to `google_gke_hub_membership_binding` resource ([#6216](https://github.com/hashicorp/terraform-provider-google-beta/pull/6216))

Reviewer: ```suggestion
* gkehub: added `labels` fields to `google_gke_hub_membership_binding` resource ([#6216](https://github.com/hashicorp/terraform-provider-google-beta/pull/6216))
```

> * GKEHUB: added `labels` fields to `google_gke_hub_membership_binding` resource ([#6216](https://github.com/hashicorp/terraform-provider-google-beta/pull/6216))

Author: ```suggestion
```

* bigquery: added `table_constraints` field to `google_bigquery_table` resource ([#6250](https://github.com/hashicorp/terraform-provider-google-beta/pull/6250))
* compute: added internal IPV6 support for `google_compute_address` and `google_compute_instance` resources ([#6232](https://github.com/hashicorp/terraform-provider-google-beta/pull/6232))
* containerattached: added `binary_authorization` field to `google_container_attached_cluster` resource ([#6256](https://github.com/hashicorp/terraform-provider-google-beta/pull/6256))
* firestore: added `point_in_time_recovery_enablement` field to `google_firestore_database` resource ([#6239](https://github.com/hashicorp/terraform-provider-google-beta/pull/6239))
* firestore: added `update_time` and `uid` fields to `google_firestore_database` resource ([#6257](https://github.com/hashicorp/terraform-provider-google-beta/pull/6257))
* gkehub2: added `labels`, `namespace_labels` fields to `google_gke_hub_namespace` resource ([#6202](https://github.com/hashicorp/terraform-provider-google-beta/pull/6202))

> * gkehub: added `labels` fields to `google_gke_hub_scope` resource ([#6243](https://github.com/hashicorp/terraform-provider-google-beta/pull/6243))

Author: ```suggestion
* gkehub: added `labels` fields to `google_gke_hub_membership_binding` resource ([#6216](https://github.com/hashicorp/terraform-provider-google-beta/pull/6216))
* gkehub: added `labels` fields to `google_gke_hub_scope` resource ([#6243](https://github.com/hashicorp/terraform-provider-google-beta/pull/6243))
```

> * gkeonprem: added `upgrade_policy` and `binary_authorization` fields in `google_gkeonprem_bare_metal_cluster` resource (beta) ([#6224](https://github.com/hashicorp/terraform-provider-google-beta/pull/6224))

Reviewer: ```suggestion
* gkeonprem: added `upgrade_policy` and `binary_authorization` fields in `google_gkeonprem_bare_metal_cluster` resource ([#6224](https://github.com/hashicorp/terraform-provider-google-beta/pull/6224))
```

> * gkeonprem: added `upgrade_policy` field in `google_gkeonprem_vmware_cluster` resource (beta) ([#6224](https://github.com/hashicorp/terraform-provider-google-beta/pull/6224))

Reviewer: ```suggestion
* gkeonprem: added `upgrade_policy` field in `google_gkeonprem_vmware_cluster` resource ([#6224](https://github.com/hashicorp/terraform-provider-google-beta/pull/6224))
```

* secretmanager: added `auto` field to `google_secret_manager_secret` resource ([#6237](https://github.com/hashicorp/terraform-provider-google-beta/pull/6237))
* secretmanager: added `deletion_policy` field to `google_secret_manager_secret_version` resource ([#6252](https://github.com/hashicorp/terraform-provider-google-beta/pull/6252))
* storage: supported in-place update for `autoclass` field in `google_storage_bucket` resource ([#6233](https://github.com/hashicorp/terraform-provider-google-beta/pull/6233))
* vertexai: added `public_endpoint_enabled` to `google_vertex_ai_index_endpoint` ([#6208](https://github.com/hashicorp/terraform-provider-google-beta/pull/6208))

> * workstations: added `env` field to `google_workstations_workstation` resource (beta) ([#6258](https://github.com/hashicorp/terraform-provider-google-beta/pull/6258))

Reviewer: ```suggestion
* workstations: added `env` field to `google_workstations_workstation` resource ([#6258](https://github.com/hashicorp/terraform-provider-google-beta/pull/6258))
```

BUG FIXES:

> * bigquerydatatransfer: fixed a bug when importing 'location' of 'google_bigquery_data_transfer_config' ([#6203](https://github.com/hashicorp/terraform-provider-google-beta/pull/6203))

Reviewer: ```suggestion
* bigquerydatatransfer: fixed a bug when importing `location` of `google_bigquery_data_transfer_config` ([#6203](https://github.com/hashicorp/terraform-provider-google-beta/pull/6203))
```

> * container: fix a bug where `additional_pod_network_configs` was not sent correctly in `google_container_node_pool` ([#6211](https://github.com/hashicorp/terraform-provider-google-beta/pull/6211))

Reviewer: ```suggestion
* container: fixed a bug where `additional_pod_network_configs` was not sent correctly in `google_container_node_pool` ([#6211](https://github.com/hashicorp/terraform-provider-google-beta/pull/6211))
```
